release_notes/370-release-notes.adoc

```diff
@@ -0,0 +1,185 @@
+
+[id="integration-ecs-rn"]
+=== Autogeneration of Amazon ECR registry integrations
+
+Registry integrations for Amazon Elastic Container Registry (ECR) are now auto-generated for Amazon Web Services (AWS) clusters. This feature requires that the nodes' Instance Identity and Access Management (IAM) Role has been granted access to ECR. You can turn off this feature by disabling the EC2 instance metadata service in your nodes. See _Amazon ECR integrations_ in the {product-title-short} Integrating documentation for more information.
+
+[id="identify-missing-policies-rn"]
+=== Identifying missing Kubernetes network policies
+
+Kubernetes network policies are vital in helping to enable zero-trust networking within a cluster. They reduce the impact of network attacks by limiting the opportunity for lateral movement. By default, Kubernetes resources are not isolated. Applying network policies is a recommended best practice left to the end user.
+
+{product-title-short} 3.70 ships with a new default policy that allows you to easily identify deployments that are not restricted by any ingress network policy and to trigger violation alerts accordingly.
+
+- The default policy is named *Deployments should have at least one ingress Network Policy*. It is disabled by default.
+
+- This default policy uses a new policy criterion called *Alert on missing ingress Network Policy*.
+
+- To identify pod isolation gaps, you can clone this default policy or create a new one by using the policy criterion and enabling it on selected resources.
+
+[id="spring-vulnerabilities"]
+=== Identifying Spring critical vulnerabilities
+
+A policy to detect Spring Cloud Function RCE vulnerability (CVE-2022-22963) and Spring Framework Spring4Shell RCE vulnerability (CVE-2022-22965) has been added. It is associated with the Critical severity and is enabled by default.
+
+[id="allow-privilege-escalation-rn"]
+=== Improved validation of pod security context
+
+A new policy criterion has been added to validate the value of `allowPrivilegeEscalation` within the Kubernetes security context. You can use this policy criterion to provide alerts when a deployment is configured to allow a container process to gain more privileges than its parent process.
+
+[id="admin-credentials-link-ocp"]
+=== Finding ACS admin credentials easily in the {ocp} console
+
+For customers deploying {product-title-short} to {ocp} using the recommended Operator method, {product-title-short} admin credentials are now directly visible from the {ocp} console. When viewing the Central object, the *Details* tab provides a clickable link to the credentials under *Admin Password Secret Reference*. The displayed credentials are the default generated password or a previously-configured and stored custom secret. See xref:../installing/install-ocp-operator.adoc#verify-central-install-operator_install-ocp-operator[Verifying Central installation] for more information.
```

Comment:
usually a capitalized item in front of the word "object" is something like an API object and gets formatted as monospaace. Not sure about this one, but thought I should mention.

Remove hyphen for "previously configured" per style guide :)

```suggestion
For customers deploying {product-title-short} to {ocp} using the recommended Operator method, {product-title-short} admin credentials are now directly visible from the {ocp} console. When viewing the `Central` object, the *Details* tab provides a clickable link to the credentials under *Admin Password Secret Reference*. The displayed credentials are the default generated password or a previously configured and stored custom secret. See xref:../installing/install-ocp-operator.adoc#verify-central-install-operator_install-ocp-operator[Verifying Central installation] for more information.
```

Reply:
Apparently it's not normally formatted as monospace, although when I made changes to the Install documentation recently, I formatted it as monospaced without knowing that. I'll go back and fix those so it's all consistently not formatted as monospace. (734 instances of no monospace vs. 12 monospaced. oops.)